app/controllers/admin/contacts_controller.rb

```diff
@@ -0,0 +1,37 @@
+class Admin::ContactsController < AdminController
+  before_action :set_contact, only: %i[show edit destroy]
+
+  def index
+    @contact = Contact.new
+    @contacts = Contact.all
+  end
+
+  def edit; end
+
+  def show; end
+
+  def create
```

Comment:
do you need to create contacts on admin/contacts page? Or this page only for displaying users contact messages?